Cypress/cypress/integration/Parallel_Tests/Create_Workflow/template.spec.js

```diff
@@ -0,0 +1,119 @@
+/// <reference types="Cypress" />
+import * as workflows from "../../../fixtures/Workflows.json";
+import * as user from "../../../fixtures/Users.json";
+
+describe("Testing the workflow creation wizard using Templates", () => {
+	before("Clearing the Cookies and deleting the Cookies", () => {
+		cy.requestLogin(user.AdminName, user.AdminPassword);
+		cy.waitForCluster("Self-Agent");
+		cy.visit("/create-workflow");
+	});
+
+	it("Running PreDefined Workflow", () => {
+		cy.chooseAgent(0);
+		cy.get("[data-cy=ControlButtons] Button").eq(0).click();
+		cy.chooseWorkflow(0, 0);
```

Comment:
```suggestion
		cy.chooseWorkflow(1, 0);
```

Reply:
@S-ayanide  Here in this step, we are first creating a workflow from pre-defined workflow and after that we are creating a template from there. Hence, I'm using the 0th index, which is for the pre-defined workflow!